<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 3.2.2 (Python 3.9.0 on win32)" generated="20210120 09:59:55.589" rpa="false">
<suite id="s1" name="OpenEmrProject" source="C:\Users\CTEA\Testing\Robot_WorkspaceTesting\OpenEmrProject">
<suite id="s1-s1" name="LoginSuite" source="C:\Users\CTEA\Testing\Robot_WorkspaceTesting\OpenEmrProject\LoginSuite">
<suite id="s1-s1-s1" name="Verify Invalid Credentials" source="C:\Users\CTEA\Testing\Robot_WorkspaceTesting\OpenEmrProject\LoginSuite\verify_invalid_credentials.robot">
<test id="s1-s1-s1-t1" name="TC3">
<status status="FAIL" starttime="20210120 10:00:00.236" endtime="20210120 10:00:00.237" critical="yes">Test case contains no keywords.</status>
</test>
<doc>This file is test for invalid Credential Testing</doc>
<status status="FAIL" starttime="20210120 09:59:55.683" endtime="20210120 10:00:00.238"></status>
</suite>
<status status="FAIL" starttime="20210120 09:59:55.678" endtime="20210120 10:00:00.241"></status>
</suite>
<status status="FAIL" starttime="20210120 09:59:55.641" endtime="20210120 10:00:00.242"></status>
</suite>
<statistics>
<total>
<stat pass="0" fail="1">Critical Tests</stat>
<stat pass="0" fail="1">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" id="s1" name="OpenEmrProject">OpenEmrProject</stat>
<stat pass="0" fail="1" id="s1-s1" name="LoginSuite">OpenEmrProject.LoginSuite</stat>
<stat pass="0" fail="1" id="s1-s1-s1" name="Verify Invalid Credentials">OpenEmrProject.LoginSuite.Verify Invalid Credentials</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20210120 10:00:00.235" level="ERROR">Calling method '_start_suite' of listener 'DataDriver' failed: XLRDError: Excel xlsx file; not supported</msg>
</errors>
</robot>
